ibm_catalog.json

```diff
@@ -116,9 +182,182 @@
             "descriptions": "",
             "features": [
               {
-                "title": "Works with all three landing zone variations (VPC, OpenShift and VSI)",
+                "title": "Supports OpenVPN Clients",
+                "description": "Yes"
+              },
+              {
+                "title": "Separate IAM access list for VPN users",
                 "description": "Yes"
               },
+              {
+                "title": "Separate VPC subnet for VPN Server",
+                "description": "Yes"
+              },
+              {
+                "title": "ACL rule that grants all access from any source",
+                "description": "Yes"
+              },
+              {
+                "title": "Security group that allows all incoming request from any source",
+                "description": "Yes"
+              },
+              {
+                "title": "IBM-managed VPN Server",
+                "description": "Yes"
+              },
+              {
+                "title": "Access to private network via VPN clients",
+                "description": "Yes"
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": "Client-to-site VPN configuration for accessing a VPC",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/main/reference-architectures/reference-architectures/cts-quickstart-da.svg",
+                  "type": "image/svg+xml"
+                },
+                "description": "Client-to-site VPN configuration for accessing a VPC."
+              }
+            ]
+          }
+        },
+        {
+          "label": "Standard",
+          "name": "standard",
+          "install_type": "fullstack",
+          "working_directory": "solutions/standard",
+          "dependencies": [
```

Comment:
It also has a dependency on the secrets manager DA

Reply:
updated